Sources/ApollonEdit/Views/Create/CreateElement/ElementCreator.swift

```diff
@@ -66,9 +66,7 @@ struct AbstractClassOrInterfaceCreator: ElementCreator {
         let element = UMLElement(id: elementID, type: type, bounds: Boundary(x: pointToAdd.x, y: pointToAdd.y, width: 200, height: 130))
         let elementAttribute = UMLElement(name: "+ attribute: Type", type: .classAttribute, owner: elementID, bounds: Boundary(x: pointToAdd.x, y: pointToAdd.y + 50, width: 200, height: 40))
         let elementMethod = UMLElement(name: "+ method()", type: .classMethod, owner: elementID, bounds: Boundary(x: pointToAdd.x, y: pointToAdd.y + 90, width: 200, height: 40))
```

Comment:
If I'm not mistaken, these variables aren't being used anymore, so maybe we can remove them? Please double check though whether this has side effects ;)
```suggestion
```